package-test/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you remove this whole folder and amend it to the previous commit? We don't need this. :)